docs/java/tips-and-tricks/common-problems.md

```diff
@@ -0,0 +1,67 @@
+---
+id: sdk-java-common-problems
+title: Some common problems and how to address them
+hide_title: true
+hide_table_of_contents: false
+sidebar_label: Frequently Debugged Problems
```

Comment:
```suggestion
sidebar_label: Troubleshooting frequent problems

```

Reply:
My idea is to make it shorter but convey the notion of finding possible solutions to known problems/difficulties/issues etc. 
I checked a couple of docs resources like google's cloud API, Amazon, etc. And this would sound just about right. We can split it into troubleshooting categories when it grows. I did similar things with FAQ.